### DISCUSSION TITLE
Official sources for building footprints

### DISCUSSION THREAD

**Discussion by tomalrussell** [OvertureMaps/data#111] [Discussion]
I wonder if some national/regional/city official footprint sources would complement the ML and crowdsourced datasets?

For example, in Europe, INSPIRE and related directives encourage openness and reusability of various datasets, including building footprints, addresses and cadastral parcels.

The [EUBUCCO](https://eubucco.com/) research project is the broadest dataset I'm aware of that brings together official open buildings data across the EU27+Switzerland, supplemented by some data under slightly restrictive licenses (CC-BY-NC/SA) and some from OpenStreetMap.

**Comment by jwass** [comment]
Agreed. We do include official footprints from a few cities in the US.

You can get the full list from an Athena query like this:
```
SELECT DISTINCT
    t.source.dataset
FROM overture_2023_12_14_alpha_0
CROSS JOIN UNNEST(sources) AS t(source)
WHERE
    theme = 'buildings'
```

which returns 
```
 dataset                                             
-----------------------------------------------------
 OpenStreetMap                                       
 Microsoft ML Buildings                              
 Google Open Buildings                               
 Esri Community Maps                                 
 USGS Lidar                                          
 Denver Regional Council of Governments 2D Buildings 
 Portland Building Footprint 2D Buildings            
 Austin Building Footprints Year 2013 2D Buildings   
 Washington DC Open Data 3D Buildings                
 Miami-Dade County Open Data 3D Buildings            
 NYC Open Data 3D Buildings                          
 Boston BPDA 3D Buildings                            
 City of Cambridge, MA Open Data 3D Buildings        
```
Lidar is for heights.

If there are any specific datasets that you think we should incorporate, let us know. Looks like EUBECCO is itself aggregating from other sources similar to Overture (including OSM) so I think we'd want to use the underlying raw data rather than the processed data.

**Reply by tomalrussell** [reply]
Thanks - I hadn't realised those US city sources were included.

EUBUCCO list their sources in a [spreadsheet](https://api.eubucco.com/v0.1/files/cc5b1b47-e1fc-4887-8a26-d8878648c3df/download). Agreed that you would likely want to go upstream and review each.

The official entry point, that may be more up-to-date, would be the [INSPIRE geoportal](https://inspire-geoportal.ec.europa.eu/srv/eng/catalog.search#/overview?view=themeOverview&theme=bu) plus [swisstopo](https://www.swisstopo.admin.ch/en/landscape-models) for Switzerland.

As a single suggestion of a place to start, France's [BDTOPO](https://geoservices.ign.fr/bdtopo) might be interesting. The footprints come with several attributes, including height, number of floors, wall and roof materials, and coarse classification of building use. IGN (the national mapping agency) appears to publish updates quite frequently (perhaps annually since 2019, quarterly since 2022-12).

**Reply by wagnerfe** [reply]
Just some additional info on EUBUCCO data: I fully agree that adding raw datasets of official sources is the best option, however the data for EUBUCCO came from 52 different sources and processing was very time consuming. (While some datasets were very easy to bulk download (e.g. [Netherlands](https://3dbag.nl/en/download)), for others we had to build webscrapers, contact the data owners via email (in one case even via a physical letter?!), reconstruct from different tiles and build the right parsers (to process .shp, .gml, .xml))

**Comment by landmaxcanada** [comment]
I am also interested in making government sourced data available, as it may be more current and accurate. I work in Canada and would be more than willing to help curate sources. What would be the best way to do this so the merged data set is the most accurate? Another source of logic would be to access the tax parcel cadastral data, which often times has both land,building values and sometimes size. So if the tax department thinks a parcel has a building, but there is no building in the main data set, then that parcel could be flagged for further analysis. Cheers James

**Reply by ttomasz** [reply]
Similarly I can help with Polish official dataset for buildings. They recently published them in parquet format which is much easier to consume than GML.
